src/UWP/ArcGISRuntime.UWP.Viewer.sln

```diff
@@ -14,6 +14,7 @@ EndProject
 Global
 	GlobalSection(SharedMSBuildProjectFiles) = preSolution
 		..\ArcGISRuntime.Samples.Shared\ArcGISRuntime.Samples.Shared.projitems*{0ad85845-0ba1-4dc9-8c32-e6fd7632e32d}*SharedItemsImports = 13
+		..\ArcGISRuntime.Samples.Shared\ArcGISRuntime.Samples.Shared.projitems*{6820b615-c568-4a13-b026-a77cdd348bfa}*SharedItemsImports = 4
```

Comment:
I don't know why this is here.

Reply:
Microsoft doesn't really know either: https://github.com/Microsoft/msbuild/issues/545

Reply:
@fragmer I was planning on reverting this change. Do you see any problems with that?

Reply:
I'd just be weary of removing VS-generated changes to sln/csproj files, in case it breaks something in a subtle/obscure way. You'd be on the hook to test building every configuration for every platform in every VS version if you revert this ;)